src/field.asm

```diff
@@ -2148,12 +2171,15 @@ FieldDelay::
 
     ; Update the combo counter.
     ldh a, [hLineClearCt]
-    ld b, a
+    dec a
```

Comment:
This doesn't seem needed. Why are you doing this? At this point we already know hLineClearCt is ≥1